test/cli/dbt-generate-profile/exists-overwrite-profile/out/profiles.yml

```diff
@@ -0,0 +1,12 @@
+Profile1:
+    outputs:
+        target1:
+            account: '{{ env_var("DBT_KBC_TARGET1_ACCOUNT") }}'
+            database: '{{ env_var("DBT_KBC_TARGET1_DATABASE") }}'
+            password: '{{ env_var("DBT_KBC_TARGET1_PASSWORD") }}'
+            schema: '{{ env_var("DBT_KBC_TARGET1_SCHEMA") }}'
+            type: '{{ env_var("DBT_KBC_TARGET1_TYPE") }}'
+            user: '{{ env_var("DBT_KBC_TARGET1_USER") }}'
+            warehouse: '{{ env_var("DBT_KBC_TARGET1_WAREHOUSE") }}'
+    target: target1
+send_anonymous_usage_stats: false
```

Comment:
3. Přepsaný původní profil (viz. https://github.com/keboola/keboola-as-code/blob/406859614b4d891ecd33a5243b96f0137c551b08/test/cli/dbt-generate-profile/exists-overwrite-profile/in/profiles.yml kde je target `target3`)